test/subdir.jl

```diff
@@ -321,4 +321,16 @@ end
     end
 end
 
+@testset "dev a package added by subdir" begin
+    isolate(loaded_depot=false) do
+        Pkg.add(url="https://github.com/DilumAluthge/FooBar.git", subdir="myfolder/Foo")
```

Comment:
I know this is not the only occurrence but it seems unfortunate to depend on an external repository, especially when the testset above has a local subdir repository set up.

Reply:
Yeah. I can update it after the tests pass.